### DISCUSSION TITLE
Install multiple software

### DISCUSSION THREAD

**Discussion by fate1ess** [microsoft/winget-cli#1173] [Q&A]
How do I install more than one software at a time?

**Comment by denelon** [comment]
@pseadev you can use `winget import` to install multiple packages with a single command.

You can `winget export packages.json` to generate a file with packages on your machine (and in the community repository). From there, you may edit the file to add or remove other packages you would like included (or not).

The command `winget import -?` displays help for import and the command `winget export -?` displays help for export.
